interpreter/exec/eval.ml

```diff
@@ -220,25 +220,26 @@ let rec step (c : config) : config =
           in v :: vs', []
         with exn -> vs', [Trapping (memory_error e.at exn) @@ e.at])
 
-      | SimdLoadLane {offset; ty; sz; _}, V128 v128 :: I32 i :: vs' ->
+      | SimdLoad {offset; ty; sz; _}, I32 i :: vs' ->
         let mem = memory frame.inst (0l @@ e.at) in
         let addr = I64_convert.extend_i32_u i in
         (try
           let v =
             match sz with
-            | None -> assert false
-            | Some (pack_size, laneidx) -> Memory.load_simd_lane v128 pack_size mem addr offset ty laneidx
+            | None -> Memory.load_value mem addr offset ty
+            | Some (pack_size, simd_load) -> V128 (Memory.load_simd_packed pack_size simd_load mem addr offset ty)
```

Comment:
Nit: line break?

Reply:
Thanks. Slight off-topic, would having some sort of formatter run on our input files be valuable? That would help codify and catch some of these stylistic issues.
I recognize that the big problem might be tons of merge conflicts when merging spec back into the proposals though...

Reply:
Yeah, unfortunately, with all the active forks, we cannot make structural changes to the sources without creating a lot of pain for a lot of people. That also prevents some refactoring in the interpreter that I'd like to do (like pulling symbol resolution out of the parser).